Packs/NationalVulnerabilityDatabaseFeed/.pack-ignore

```diff
@@ -0,0 +1,2 @@
+[file:NationalVulnerabilityDatabaseFeed.py]
+ignore=E731
```

Comment:
It related only to validation errors.
In order to ignore lint errors, we use `# noqa: <error_code>` in the line that provide the error.